controllers/managedclusterForCGU_controller.go

```diff
@@ -156,8 +157,13 @@ func (r *ManagedClusterForCguReconciler) newClusterGroupUpgrade(ctx context.Cont
 	for _, policy := range policies.Items {
 		deployWave, found := policy.GetAnnotations()[ztpDeployWaveAnnotation]
 		if found {
+			deployWaveInt, err := strconv.Atoi(deployWave)
+			if err != nil {
+				// err convert from string to int
+				return err
```

Comment:
It would be helpful to return an error which indicates that in policy xxxxxx the ran.openshift.io/ztp-wave-deploy value is not an integer.

Reply:
Done